avocado/core/task/statemachine.py

```diff
@@ -247,6 +247,7 @@ async def start(self):
             await asyncio.sleep(0.1)
             return
 
+        await self._spawner.create_task_output_dir(runtime_task)
```

Comment:
I'm not sure if this is the right place for this call. IIUC, the statemachine would be responsible only for the 'state machine', having this logic inside this file would open a precedence to more code living here. I'm not sure, but would be nice to double check.

@clebergnu ?